data/json/npcs/refugee_center/surface_refugees/NPC_Draco_Dune.json

```diff
@@ -81,28 +107,26 @@
         "switch": true
       },
       {
-        "text": "Hi, Draco.  What's up?",
-        "condition": { "days_since_cataclysm": 180 },
-        "topic": "TALK_REFUGEE_Draco_2_stub",
-        "switch": true
+        "text": "Hi, Draco, how are you doing?",
```

Comment:
consider making this part of the existing switch, which will simplify the logic.

Reply:
Not sure what you mean, but other requested changes have been made. The weed sale doesn't work now though.

Reply:
bump